src/Controls/samples/Controls.Sample/XamlApp.xaml

```diff
@@ -0,0 +1,11 @@
+﻿<Application xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:MauiSampleApp"
+             xmlns:maui="clr-namespace:Microsoft.Maui;assembly=Microsoft.Maui"
+             x:Class="MauiSampleApp.XamlApp">
+    <Application.Resources>
+        <ResourceDictionary>
+            <maui:Color x:Key="DotNetPurple">#512BD4</maui:Color>
```

Comment:
Here is the XAML and Resources that David wanted.

Reply:
Is the maui namespace b/c we haven't updated the base xmlns yet?

Reply:
I think we going to have both no? Add a new one for MAUI, but the old one also points to Maui.Controls